index.bs

```diff
@@ -1655,7 +1655,7 @@ This method has no arguments and returns a Boolean value.
         authenticator by the client in its call to either {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}.
 </div>
 
-### Information about Public Key Credential (interface <dfn interface>AuthenticatorAttestationResponse</dfn>) ### {#iface-authenticatorattestationresponse}
+### Information About Public Key Credential (interface <dfn interface>AuthenticatorAttestationResponse</dfn>) ### {#iface-authenticatorattestationresponse}
```

Comment:
POI: "About"

Reply:
POI: I did not capitalize the parenthesized "interface" here, or the "enum" or "dictionary" in similar headings.